.gitignore

```diff
@@ -5,4 +5,5 @@
 *.pyc
 *tmp*
 *.egg-info
-_quest.c
\ No newline at end of file
+_quest.c
+pyquest_cffi/questlib/*
```

Comment:
The whole questlib folder should not be in .gitignore. The __init__.py, _abstract.py and build_quest.py files in there are not generated automatically. You could use .../questlib/*.o .../questlib/*.c .../questlib/*.dylib